cartridge-cli/VERSION.lua

```diff
@@ -1 +1 @@
-return '1.3.2'
+return '1.4.0'
```

Comment:
Why not 1.3.3?

Reply:
Because there are new features - `cartridge build` and build directory specifying.